tests/unit/color_picker.coffee

```diff
@@ -48,3 +48,16 @@ test 'Color picker converts color to hex when color is invalid', ->
   testHexConversion "foo", undefined
   testHexConversion "rgb(a, b, c)", undefined
   testHexConversion "rgb(1,2,3)", undefined
+
+test 'Custom selected color is set as custom color', ->
+  color = '#addec0'
+  colorPicker.set 'selectedColor', color
+  equal colorPicker.get('customColor'), color
+
+test 'Clicking color picker cell sets selectedColor', ->
```

Comment:
You're using the `click` event. You need an event dispatcher in your test setup.

See https://github.com/Addepar/ember-widgets/blob/master/tests/functional/multi_select_test.coffee#L10-L11